### DISCUSSION TITLE
Some Error form Opacity

### DISCUSSION THREAD

**Discussion by lymtu** [tailwindlabs/tailwindcss#13490] [Help]
![code](https://github.com/tailwindlabs/tailwindcss/assets/134980038/17fa27d7-5e5d-40a5-9479-dc36160edd1e)

It will be displayed when the opacity-80 property is set for the second layer child element div. opacity-100 will not be displayed when set; Not displayed when bg-opacity-60 is set, too.

**Comment by alex-krasikau** [comment]
Hey @lymtu!

If you have conflicting utilities set on the same element, you can't guarantee which one will take priority in production. To avoid this, wrap your class string with [Tailwind Merge](https://github.com/dcastil/tailwind-merge), it merges the class strings and makes sure that the last declaration will be applied to the element. Your code will then look like this:

```
import { twMerge } from 'tailwind-merge'

...
<div className={twMerge(class_, "col-span-2 row-span-2")}>
...
```

**Reply by lymtu** [reply]
Thanks,let me try!
